src/integration-tests/commonTests.js

```diff
@@ -12,46 +13,45 @@ const logIn = async () => {
 		let opts = {
 			resources: [baseURL],
 			delay: 1000, // initial delay in ms, default 0
-			interval: 500, // poll interval in ms, default 250ms
-			timeout: 30000,
+			interval: 1000, // poll interval in ms, default 250ms
+			timeout: 60000,
 		}
 		await waitOn(opts)
-		let email = 'nevil' + crypto.randomBytes(5).toString('hex') + '@tunerlabs.com'
+		let firstname = 'adithya'
+		let email = firstname + crypto.randomBytes(5).toString('hex') + '@tunerlabs.com'
 		let password = faker.internet.password()
 		let res = await request.post('/user/v1/account/create').send({
-			name: 'Nevil',
+			name: firstname,
```

Comment:
@adithyadinesh0412 directly assign the data first name, so don't want to assign variable